src/style.css

```diff
@@ -0,0 +1,129 @@
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+/* #region Stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars .stars__star:last-child {
+  margin-right: 0;
+}
+/* #endregion */
+
+/* #region Card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px 4px;
+}
+
+.card__description {
+  font-weight: 400;
+  color: #616070;
+  font-size: 10px;
+  margin: 0 16px 16px;
+
+}
+
+.card__ratings {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 16px 24px;
+
+}
+
+.card__reviews {
+  font-weight: 400;
+  color: #060b35;
+  font-size: 10px;
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 16px;
+}
+
+.card__price-title {
+  font-weight: 400;
+  color: #616070;
+  margin: 0;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  background-color: #00acdc;
+  margin: 0 16px 16px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  transition: 0.3s;
```

Comment:
you need to specify which properties you set the transition for. look more into the syntax